lib/routes/shared/initial_walkthrough.dart

```diff
@@ -43,6 +49,7 @@ class InitialWalkthroughPageState extends State<InitialWalkthroughPage>
     _instructions = widget._isPos ?
     "The simplest, fastest & safest way\nto earn bitcoin" :
     "The simplest, fastest & safest way\nto spend your bitcoins";
+    _updateSecurityModelAction = UpdateSecurityModel(widget._user.securityModel.copyWith(backupKeyType: BackupKeyType.NONE));
```

Comment:
Better not save this as class variable, just create it on `initState` when needed.

Reply:
It's used on multiple streams. It's value is being set on multipleRestoreStream and used after successful restore on restoreFinishedStream.